streaming/streaming.go

```diff
@@ -169,6 +175,21 @@ func dataStreamGaxRetrier() gax.Retryer {
 	})
 }
 
+// Returns ExponentialBackoff Operator for DataStream Update Stream Retries across LRO poll.
+func getUpdateDataStreamLRORetryBackoff() *backoff.ExponentialBackOff {
+	dataStreamUpdateBackoff := &backoff.ExponentialBackOff{
+		InitialInterval:     DEFAULT_DATASTREAM_LRO_POLL_BASE_DELAY,
+		RandomizationFactor: backoff.DefaultRandomizationFactor,
+		Multiplier:          DEFAULT_DATASTREAM_LRO_POLL_MULTIPLIER,
+		MaxInterval:         DEFAULT_DATASTREAM_LRO_POLL_MAX_DELAY,
+		MaxElapsedTime:      DEFAULT_DATASTREAM_LRO_POLL_MAX_ELAPSED_TIME,
+		Stop:                backoff.Stop,
+		Clock:               backoff.SystemClock,
+	}
+	dataStreamUpdateBackoff.Reset()
+	return dataStreamUpdateBackoff
+}
```

Comment:
Why do we need a separate configuration for LRO exponential backoff retry and normal API retries?

Reply:
The configuration parameters for retries when an API call errors out could be quite different than the retry when an LRO errors out and we should not tightly couple them.
For example an API could be retried after an initial delay of say 300 milliseconds and ramped up to an amount with exponential backoff, an LRO poll typically takes a few minutes to complete, so the initial retry could be in minutes instead of milliseconds.

Reply:
We have most of the basic stuff needed to: 
1. Add accessor for LaunchStream
2. Rerfactor launchsetream and add ut.

I need to think through 2 things though:
1. Should we have a single accessor for laucnh stream (this involves both create and update calls)
2. Should we have 2 separate accessors
3. Should resourcemanager accessor be chained from outside or should the call to reasourcemanager accessor be made outside and only output passed to the datastream_accessor.

This is the place (making 2 API calls for 1 client and 1 API call for another client) where we probably desperately need a DI framework :(